website/pages/blog/defer-support-codegen.mdx

```diff
@@ -170,6 +170,15 @@ export default App;
 
 With the above code, the `orders` field is rendered only if it's available in the data object.
 
+### How Does It Compare with Relay?
+
+Relay allows for incremental delivery of data from the server to the client without an extra logic
+on the client because it uses Suspense. It enables the client to render parts of the UI as soon as
+the corresponding data is available.
+
+The same can be achieved in other GraphQL clients with the `@defer` directive and the
+`isFragmentReady` utility function.
```

Comment:
Maybe also adds sth like

"Because using suspense requires deep integration with the graphql client code, graphql code generator does not support this out of the box. We are looking forward to collaborating with both the maintainers of apollo-client, urql, and co for a more generic suspense-backed implementation in the future".